streampipes-processors-filters-jvm/src/main/resources/org.apache.streampipes.processors.filters.jvm.merge/documentation.md

```diff
@@ -38,12 +38,11 @@ The following figure shows how the events of the two data streams will be mergrg
 
 ## Required input
 Each of the data streams needs a timestamp.
-***
```

Comment:
@mohanvive Do we have to remove this? If so, some documentation files still seem to have `***` under required-input sections. We might have to remove them as well.

Reply:
It is a mistake. I have fixed it now. Thanks.